includes/html/pages/iftype.inc.php

```diff
@@ -19,10 +19,14 @@
 
 // show title from config file (but ucwords it)
 $ctypes = collect(\LibreNMS\Config::get('custom_descr', []))->keyBy(function ($descr) {
+    if (is_array($descr)) {
+        return strtolower($descr[0]);
+    }
+
     return strtolower($descr);
 });
 array_walk($types_array, function (&$type) use ($ctypes) {
-    $type = ucwords($ctypes->get(strtolower($type), $type));
+    $type = ucwords($ctypes->get(strtolower($type[0]), $type));
```

Comment:
This is not right, $type is a string here.

$ctypes->get(strtolower($type), $type) will return the array of your config, so maybe something like this?
```suggestion
    $name = $ctypes->get(strtolower($type), $type);
    $type = ucwords($name[0] ?? $name);
```